drydock/templates/drydock/k8s/ingress/issuer.yml

```diff
@@ -0,0 +1,23 @@
+{% if DRYDOCK_AUTO_TLS -%}
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt
+  namespace: {{ K8S_NAMESPACE }}
+  labels:
+    app.kubernetes.io/name: letsencrypt
+spec:
+  acme:
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: {{ DRYDOCK_LETSENCRYPT_EMAIL|default(CONTACT_EMAIL) }}
```

Comment:
`DRYDOCK_LETSENCRYPT_EMAIL` variable is not in the plugin defaults. Should we add it?

Reply:
If we define any default value, it will take this instead of CONTACT_EMAIL, in older versions of drydock we don't have this either

Reply:
It's strange to support a variable not referenced anywhere, not in the defaults or the docs; nobody will set it. I would set `DRYDOCK_LETSENCRYPT_EMAIL` in the plugin defaults and as its default value I'll set `CONTACT_EMAIL`

Reply:
`CONTACT_EMAIL` does not exists in the context of the plugin, i set it as `None` and add `true` to the default to avoid set `DRYDOCK_LETSENCRYPT_EMAIL` when is `None`